### ISSUE TITLE
Cant compile source code

### ISSUE THREAD

**Issue by haoneRG** [ShadowMario/FNF-PsychEngine#12813]
### Describe your problem here.

i installed hmm and everything that it needs but the console says that hxCodec is not installed, even tho it is.
![image](https://github.com/ShadowMario/FNF-PsychEngine/assets/123265605/0e68b9fc-58dc-49d9-99d0-5b565d4f4ee8)
here my lib's
![image](https://github.com/ShadowMario/FNF-PsychEngine/assets/123265605/3f1563ae-61f5-4683-82c9-a60e0852c5b4)

### Are you modding a build from source or with Lua?

Source

### What is your build target?

Windows x64

**Comment by MemeHovy** [comment]
it's a bug with hmm

**Comment by haoneRG** [comment]
> it's a bug with hmm
what can i do?

**Comment by KutikiPlayz** [comment]
What works for me is just removing hmm and then go through the list of stuff in hmm.json and manually install them

here's the list of commands to run so you don't have to do it yourself
```
haxelib remove hmm
haxelib install lime
haxelib install openfl
haxelib install flixel
haxelib install flixel-addons
haxelib install flixel-tools
haxelib install flixel-ui
haxelib install hxcpp
haxelib install tjson
haxelib install hxjsonast
haxelib install hxCodec
haxelib install script
haxelib install hxcpp-debug-server
haxelib git discord_rpc https://github.com/Aidan63/linc_discord-rpc
haxelib git linc_luajit https://github.com/superpowers04/linc_luajit
```

haven't encountered any issues using the latest versions yet, just make sure your haxe version is 4.2.5

**Comment by haoneRG** [comment]
> What works for me is just removing hmm and then go through the list of stuff in hmm.json and manually install them
> 
> here's the list of commands to run so you don't have to do it yourself
> 
> ```
> haxelib remove hmm
> haxelib install lime
> haxelib install openfl
> haxelib install flixel
> haxelib install flixel-addons
> haxelib install flixel-tools
> haxelib install flixel-ui
> haxelib install hxcpp
> haxelib install tjson
> haxelib install hxjsonast
> haxelib install hxCodec
> haxelib install script
> haxelib install hxcpp-debug-server
> haxelib git discord_rpc https://github.com/Aidan63/linc_discord-rpc
> haxelib git linc_luajit https://github.com/superpowers04/linc_luajit
> ```
> 
> haven't encountered any issues using the latest versions yet, just make sure your haxe version is 4.2.5

didnt work here

**Comment by NexIsDumb** [comment]
> it's a bug with hmm

Nopp, it's a bug with hxCodec, fortunately it got fixed in the next version 3.0.3
For now you can use this to resolve: #12770
